docs/self-managed/installations/docker-compose-custom-certs.md

```diff
@@ -238,37 +246,27 @@ Did you encounter a problem during the installation process ? See the
 
 ## ⏫ Update
 
-Follow these steps to update your self-managed instance with the latest features
-
-1. **Update the [`self-managed`](https://github.com/r2devops/self-managed/) Git Repository**
-
-   Navigate to the location of your [`self-managed`](https://github.com/r2devops/self-managed/) repository and execute the following commands:
+Follow these steps to update your self-managed instance to a new version:
 
+1. Navigate to the location of your
+   [`self-managed`](https://github.com/r2devops/self-managed/) git repository
+1. Update it
    ```sh
-   git fetch          # Fetch the latest changes from the repository
-   git pull           # Pull the latest changes from the repository into your directory
+   git pull
    ```
-
-2. **Update Your `.env` File**
-
-   To update the `Frontend` and `Backend` versions in your `.env` file, follow these steps:
-
-   1. Open your `.env.example` file.
-   2. Locate and copy the new versions of `FRONTEND_IMAGE_TAG` and `BACKEND_IMAGE_TAG`.
-   3. Open your `.env` file and update the same variables with the new versions:
-
-   ```sh
-   FRONTEND_IMAGE_TAG="<new_frontend_version>" # For example, v1.35.0
-   BACKEND_IMAGE_TAG="<new_backend_version>" # For example, v1.39.0
+1. Open the `.env.example` file and copy the values of  `FRONTEND_IMAGE_TAG`
+   and `BACKEND_IMAGE_TAG` variables
+1. Edit the `.env` file by updating values of `FRONTEND_IMAGE_TAG` and
+   `BACKEND_IMAGE_TAG` variables with the values previously copied
+   ```sh title=".env" hl_lines="1-2"
+   FRONTEND_IMAGE_TAG="new frontend version>"
```

Comment:
`<new frontend version>` instead of `new frontend version>`

Reply:
Fixed